egret/models/tests/test_unit_commitment.py

```diff
@@ -30,7 +30,7 @@
 ## for FP comparisons
 rel_tol = 1e-08
 
-solver_list = ['gurobi_persistent', 'cplex_persistent', 'gurobi', 'cplex']
+solver_list = ['xpress_persistent', 'gurobi_persistent', 'cplex_persistent', 'gurobi', 'cplex']
```

Comment:
A nit - shouldn't 'xpress' also be included in the solver list?

Reply:
Ideally, yes. But the Xpress LP interface basically doesn't work with modern versions of Xpress -- at least not on my system. Pyomo should probably default to the direct interface, but that is still waiting to be merged.